src/collections/pages/fr/about.md

```diff
@@ -0,0 +1,18 @@
+---
+title: À propos
+permalink: fr/a-propos/
```

Comment:
You shouldn't need to specify this if `generatePermalink` is properly set up, let's discuss.